homeassistant/components/velbus/config_flow.py

```diff
@@ -64,12 +63,12 @@ def _prt_in_configuration_exists(self, prt: str) -> bool:
 
     async def async_step_import(self, user_input=None):
         """Import a config entry."""
+        user_input[CONF_NAME] = 'Velbus Import'
         prt = user_input.get(CONF_PORT)
-        name = user_input.get(CONF_NAME)
         if self._prt_in_configuration_exists(prt):
             # if the velbus import is already in the config
             # we should not proceed the import
             return self.async_abort(
                 reason='already_imported'
```

Comment:
We don't know if the existing config entry was imported or if it was set up via user. I suggest using a similar abort string as the error string when aborting due to existing config entry port, ie `'port_exists'`.

Reply:
This part is only happening in the import step so it will only run when we are importing config.

if i remove this part, the velbus component is rediscoverd every time we restart hass (as long as the config is still in the configfile)
if this is ok i can remove this check indeed

Reply:
The existing config entry might not have been imported but set up via user step. That's why we shouldn't phrase it like it's already been imported, since that might not be true.

We shouldn't remove this check, we should just use different message language and rename the message string, to make this clearer.